README.md

```diff
@@ -19,6 +19,14 @@ Run this service from your console using: `sbt run`
 
 Use service manager to start up supporting services: `sm2 --start TGP_FE` 
 
+### Testing
+
+Run this to check tests: `sbt test`
+
+Run this to check code coverage: `sbt clean coverage test`
+
+Run this to get code coverage report: `sbt coverageReport`
+
```

Comment:
Commands to lint also need to be included in here

Reply:
Added the command for lint :)